addons/point_of_sale/static/src/app/store/pos_store.js

```diff
@@ -490,6 +490,15 @@ export class PosStore extends Reactive {
     }
 
     async afterProcessServerData() {
+        // Adding the not synced paid orders to the pending orders
+        const paidUnsyncedOrderIds = this.data.models["pos.order"]
```

Comment:
```suggestion
        const paidUnsyncedOrderIds = this.models["pos.order"]
```

In the pos_store you can directly access models.